_pages/index.md

```diff
@@ -17,10 +17,21 @@ Every project needs an Authority to Operate (ATO). ATOs cover the important secu
 
 ### Outreach
 
+* Appoint a storyteller for your project early in the cycle and notify #outreach. The storyteller will be your project's conduit with the outreach team and should notify the outreach team about: 
+
+    - upcoming milestones and events (that you’re going to or could present at)
+    - any impending risks to prepare for press and mitigate any issues
+    - progress on comms plan / blog posts
+    - any other ways the comms team can help your project
+    - upcoming speaking events / conferences that the project/product is represented at
```

Comment:
this seems redundant to "upcoming milestones and events" -- maybe just add "/conferences" to that?